pennylane/ops/op_math/controlled_class.py

```diff
@@ -305,6 +306,20 @@ def pow(self, z):
             for op in base_pow
         ]
 
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1) -> "ControlledOp":
+        if depth == 0:
+            return self
+        return ControlledOp(
```

Comment:
```suggestion
        return Controlled(
```

`ControlledOp` is just for base ops that inherit from `Operation`, not just `Operator`.  We want to get rid of the distinction eventually, but for now, it's still there.

For example, what if we had a controlled projector? Or a controlled Hamiltonian?